src/server/kernel/woutputlayout.cpp

```diff
@@ -57,7 +85,17 @@ void WOutputLayout::move(WOutput *output, const QPoint &pos)
     if (output->position() == pos)
         return;
 
+    bool maybeImplicitSizeChange = false;
```

Comment:
Why not direct call `updateImplicitSize`

Reply:
not necessary